### PR TITLE
Product maturity phases

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: 1.8/overview/maturity.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+- [Communication](#communication)
+- [Current DC/OS feature maturity](#current-maturity)
+
+# <a name="criteria"></a>Criteria
+
+## Completeness
+
+**Functionality:** Completeness of the feature implementation.
+
+**Interfaces:** Feature has an API with deprecation cycle, CLI, and UI.
```
Comment (keithchambers) [inline 1.8/overview/maturity.md]: Strictly speaking not all features require a CLI and UI. We will not require CLI and UI for a feature to be considered mature.

File: 1.8/overview/maturity.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+### Quality
+
+* Limited or no functional test
+* Limited or no system test
+* Limited or no Mesosphere dogfooding
+
+### Support
+
+* No technical support from Mesosphere
```
Comment (keithchambers) [inline 1.8/overview/maturity.md]: Let's change this to `No SLA governed support available`

File: 1.8/overview/maturity.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+* Robust functional test
+* Robust performance testing
+* Robust fault testing
+* Robust Mesosphere dogfooding
+
+### Support
+
+* SLA governed support available
+
+# <a name="establishing-maturity"></a>Establishing Maturity
```
Comment (keithchambers) [inline 1.8/overview/maturity.md]: I recommend we remove this.

File: 1.8/overview/maturity.md (hunk 1 above)
Comment (joel-hamill) [inline 1.8/overview/maturity.md]: @keithchambers so should this be removed?

File: 1.8/overview/maturity.md (hunk 3 above)
Comment (joel-hamill) [inline 1.8/overview/maturity.md]: this was removed in subsequent PR.

**Pull request merged**